docs/setup/install.md

```diff
@@ -0,0 +1,151 @@
+<!-- ``` -->
+
+<!-- ### Uninstall -->
+
+<!-- To uninstall a released version of JobSet from your cluster, run the following command: -->
+
+<!-- ```shell -->
+<!-- VERSION=v0.1.0 -->
+<!-- kubectl delete -f https://github.com/kubernetes-sigs/jobset/releases/download/$VERSION/manifests.yaml -->
+<!-- ``` -->
+
+<!-- <\!-- Uncomment once we have component config setup -\-> -->
+<!-- <\!-- ## Install a custom-configured released version -\-> -->
+
+<!-- <\!-- To install a custom-configured released version of JobSet in your cluster, execute the following steps: -\-> -->
+
+<!-- <\!-- 1. Download the release's `manifests.yaml` file: -\-> -->
+
+<!-- <\!-- ```shell -\-> -->
+<!-- <\!-- VERSION=v0.1.0 -\-> -->
+<!-- <\!-- wget https://github.com/kubernetes-sigs/jobset/releases/download/$VERSION/manifests.yaml -\-> -->
+<!-- <\!-- ``` -\-> -->
+<!-- <\!-- 2. With an editor of your preference, open `manifests.yaml`. -\-> -->
+<!-- <\!-- 3. In the `jobset-manager-config` ConfigMap manifest, edit the -\-> -->
+<!-- <\!-- `controller_manager_config.yaml` data entry. The entry represents -\-> -->
+<!-- <\!-- the default JobSet Configuration -\-> -->
+<!-- <\!-- struct ([v1alpha1@v0.1.0](https://pkg.go.dev/sigs.k8s.io/jobset@v0.1.0/apis/config/v1alpha1#Configuration)). -\-> -->
+<!-- <\!-- The contents of the ConfigMap are similar to the following: -\-> -->
+
+
+<!-- <\!-- ```yaml -\-> -->
+<!-- <\!-- apiVersion: v1 -\-> -->
+<!-- <\!-- kind: ConfigMap -\-> -->
+<!-- <\!-- metadata: -\-> -->
+<!-- <\!--   name: jobset-manager-config -\-> -->
+<!-- <\!--   namespace: jobset-system -\-> -->
+<!-- <\!-- data: -\-> -->
+<!-- <\!--   controller_manager_config.yaml: | -\-> -->
+<!-- <\!--     apiVersion: config.jobset.x-k8s.io/v1alpha1 -\-> -->
+<!-- <\!--     kind: Configuration -\-> -->
+<!-- <\!--     namespace: jobset-system -\-> -->
+<!-- <\!--     health: -\-> -->
+<!-- <\!--       healthProbeBindAddress: :8081 -\-> -->
+<!-- <\!--     metrics: -\-> -->
+<!-- <\!--       bindAddress: :8080 -\-> -->
+<!-- <\!--     webhook: -\-> -->
+<!-- <\!--       port: 9443 -\-> -->
+<!-- <\!--     internalCertManagement: -\-> -->
+<!-- <\!--       enable: true -\-> -->
+<!-- <\!--       webhookServiceName: jobset-webhook-service -\-> -->
+<!-- <\!--       webhookSecretName: jobset-webhook-server-cert -\-> -->
+<!-- <\!-- ``` -\-> -->
+
+<!-- <\!-- 3. Apply the customized manifests to the cluster: -\-> -->
+
+<!-- <\!-- ```shell -\-> -->
+<!-- <\!-- kubectl apply -f manifests.yaml -\-> -->
+<!-- <\!-- ``` -\-> -->
+
+## Install the latest development version
+
+To install the latest development version of Jobset in your cluster, run the
+following command:
+
+```shell
+kubectl apply -k github.com/kubernetes-sigs/jobset/config/default?ref=main
+```
+
+The controller runs in the `jobset-system` namespace.
+
+### Uninstall
+
+To uninstall JobSet, run the following command:
+
+```shell
+kubectl delete -k github.com/kubernetes-sigs/jobset/config/default
```

Comment:
Tried manually testing these commands and for some reason, kubectl -k flag always results in a decoding error for me (same as original issue preventing CRD installation we discussed today):

`error: unable to decode "github.com/kubernetes-sigs/jobset/config/default": parsing time "null" as "2006-01-02T15:04:05Z07:00": cannot parse "null" as "2006"`

We can merge this to test the cloudbuild now but we will need to look into this I think.